README.md

```diff
@@ -3,43 +3,54 @@
 <div align="center">
 
 [![NPM version](https://img.shields.io/npm/v/env-var.svg?style=flat)](https://www.npmjs.com/package/env-var)
-[![TypeScript](https://badgen.net/npm/types/env-var)](http://www.typescriptlang.org/)
+[![Typescript](https://badgen.net/npm/types/env-var)](http://www.Typescriptlang.org/)
 [![License](https://badgen.net/npm/license/env-var)](https://opensource.org/licenses/MIT)
 [![Travis CI](https://travis-ci.org/evanshortiss/env-var.svg?branch=master)](https://travis-ci.org/evanshortiss/env-var)
 [![Coverage Status](https://coveralls.io/repos/github/evanshortiss/env-var/badge.svg?branch=master)](https://coveralls.io/github/evanshortiss/env-var?branch=master)
 [![npm downloads](https://img.shields.io/npm/dm/env-var.svg?style=flat)](https://www.npmjs.com/package/env-var)
 [![Known Vulnerabilities](https://snyk.io//test/github/evanshortiss/env-var/badge.svg?targetFile=package.json)](https://snyk.io//test/github/evanshortiss/env-var?targetFile=package.json)
 
 
-Verification, sanitization, and type coercion for environment variables in
-Node.js. Supports TypeScript!
+Verification, sanitization, and type coercion for environment variables in Node.js. Supports Typescript!
 <br>
 <br>
 </div>
 
 * 🏋 Lightweight. Zero dependencies and just ~4.7kB when minified!
-* 🧹 Cleaner code, as [shown here](https://gist.github.com/evanshortiss/0cb049bf676b6138d13384671dad750d)
-* 🚫 [Fail fast](https://en.wikipedia.org/wiki/Fail-fast) if your environment is misconfigured
-* 👩‍💻 Friendly error messages and example values improve developer experience
-* 🎉 TypeScript support provides compile time safety and better developer experience
+* 🧹 Clean and simple code, as [shown here](https://gist.github.com/evanshortiss/0cb049bf676b6138d13384671dad750d).
+* 🚫 [Fails fast](https://en.wikipedia.org/wiki/Fail-fast) if your environment is misconfigured.
+* 👩‍💻 Friendly error messages and example values for better debugging experience.
+* 🎉 Typescript support provides compile time safety and better developer experience.
+* 📦 Can be used for non-NodeJS projects such as [React and React-native projects](https://github.com/evanshortiss/env-var/pull/138).
 
+## Documentation structure
+
+- [API.md](API.md): The full API set for `env-var`
+- [CHANGELOG.md](CHANGELOG.md)
+- [CODE_OF_CONDUCT.md](CODE_OF_CONDUCT.md)
+- [CONTRIBUTING.md](CONTRIBUTING.md)
+- [EXAMPLE.md](EXAMPLE.md): Example usage of `env-var`
+- [README.md](README.md): General and key information on this project
```

Comment:
I'd remove this since we're already in the README here :)

Reply:
Removed!